<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SquidGame</class>
 <widget class="QWidget" name="SquidGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SquidGame</string>
  </property>
  <widget class="QLabel" name="label_Grid">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>610</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kristen ITC</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>STEP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_GridNum">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>590</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kristen ITC</family>
     <pointsize>120</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_StepTotal">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>600</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kristen ITC</family>
     <pointsize>120</pointsize>
    </font>
   </property>
   <property name="text">
    <string>/10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Next">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>741</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Soi">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>741</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mark">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>400</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Gothic Std</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>○ △ □</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
